pkg/controller/statefulset/stateful_set_utils.go

```diff
@@ -318,9 +328,9 @@ func newRevision(set *apps.StatefulSet, revision int64) (*apps.ControllerRevisio
 		revision)
 }
 
-// applyRevision returns a new StatefulSet constructed by restoring the state in revision to set. If the returned error
+// ApplyRevision returns a new StatefulSet constructed by restoring the state in revision to set. If the returned error
```

Comment:
To prevent writing a redundant function similar to `applyRevision`, i exposed the function. Should i do so?